bucket/redis.json

```
@@ -1,5 +1,6 @@
 {
     "homepage": "https://redis.io",
+    "description": "An open source (BSD licensed), in-memory data structure store, used as a database, cache and message broker.",
```

No need to mention license in description

```suggestion
    "description": "In-memory data structure store, used as a database, cache and message broker.",
```

---

Agreed. I'm just cut-n-pasting, but I don't think the description needs to include:
1. The name of the app (unless it's not the same as the json file).
2. Licensing
3. Open/closed source
4. Command-line or GUI (as we know it's a GUI/CLI by it having a shortcut or not).
5. Marketing fluff ("bestest ever!")